### PR TITLE
PLAT-3962 - Support usage dashboard

### PR DESCRIPTION
**Pull request by rkreich**

### CODE REVIEW

File: alpha/lib/enums/PermissionName.php [hunk 1]
```diff
@@ -236,6 +236,7 @@
 	
 	const FEATURE_SHOW_FLASH_STUDIO = 'FEATURE_SHOW_FLASH_STUDIO';
 	const FEATURE_SHOW_HTML_STUDIO = 'FEATURE_SHOW_HTML_STUDIO';
+	const FEATURE_ENABLE_USAGE_DASHBOARD = 'FEATURE_ENABLE_USAGE_DASHBOARD';
```
Comment (tan-tan-kanarek) [inline alpha/lib/enums/PermissionName.php]: If you don't refer to this permission in the server code, there is no reason to add it here.

Comment (rkreich) [inline alpha/lib/enums/PermissionName.php]: @tan-tan-kanarek, we are not trying to keep all available permissions organized in one place?
cc @erankor

Comment (tan-tan-kanarek) [inline alpha/lib/enums/PermissionName.php]: Yes, in the DB.
This enum contains only few permissions that used internally by the server code.

Comment (rkreich) [inline alpha/lib/enums/PermissionName.php]: Removed in 5eebcb79018b2c80fa31c5e84037d83274d834db

**Pull request merged**